src/lib/components/Markdown.js

```diff
@@ -48,9 +49,14 @@ export const Markdown = (props) => {
         blockquote: ({ node, ...props }) => (
           <blockquote className="blockquote" {...props} />
         ),
-        table: ({ node, ...props }) => (
-          <table className="table table-striped" {...props} />
-        ),
+        table: ({ node, ...props }) =>
+          responsiveTable ? (
```

Comment:
A more flexible solution can be to add `onTable` (similar to `onLink` and `onImage`) instead of `responsiveTable`